docs/manual/java/guide/build/ImmutablesInIDEs.md

```diff
@@ -40,5 +40,5 @@ If you are importing many projects and you find the above configuration dialogs
 
 To setup the Immutables annotation processor in IntelliJ you need to follow [these instructions](https://immutables.github.io/apt.html#intellij-idea) (there is one caveat though, please make sure to read the next paragraph) for each project that is using the [Immutables](https://immutables.github.io) tool to generate [[immutable objects|Immutable]].
 
-The one caveat is that you should set the "Production sources directory" to `target/scala-2.11/src_managed/main` and the "Test source directory" to `target/src_managed/test`, instead of using the ones recommended in the linked instructions. The reason is that there are a couple of issues related to using immutables in IntelliJ with the IntelliJ sbt plugin (specifically, [SCL-8543](https://youtrack.jetbrains.com/issue/SCL-8543) and [IDEA-117540](  https://youtrack.jetbrains.com/issue/IDEA-117540)).
-Finally, under "Project Structure" you should undo the exclusion of the `target`-directory and verify whether `target/scala-2.11/src_managed/main` is marked as 'Sources' for each module.
+The one caveat is that you should set the "Production sources directory" to `target/scala-2.12/src_managed/main` and the "Test source directory" to `target/src_managed/test`, instead of using the ones recommended in the linked instructions. The reason is that there are a couple of issues related to using immutables in IntelliJ with the IntelliJ sbt plugin (specifically, [SCL-8543](https://youtrack.jetbrains.com/issue/SCL-8543) and [IDEA-117540](  https://youtrack.jetbrains.com/issue/IDEA-117540)).
+Finally, under "Project Structure" you should undo the exclusion of the `target`-directory and verify whether `target/scala-2.12/src_managed/main` is marked as 'Sources' for each module.
```

Comment:
These paths might be misleading to anyone who doesn't migrate to 2.12. I wonder if there's a way to rephrase this to make it clear that you need to use the right directory for the selected Scala version. Maybe we can add a note saying the directories need to be adjusted for Scala 2.11 and link to "A note on Scala versions" in the doc below?

Reply:
I'm adding the following: 
(or `target/scala-2.11/src_managed/main` if using Scala 2.11)